ansible/roles/php72/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+fpm_user: www-data
+fpm_group: www-data
+fpm_listen: 127.0.0.1:9000
+fpm_log_path: /var/log/php
+
+php_ini:
+  'date.timezone': "Europe/Berlin"
+  'memory_limit': 2048M
```

Comment:
wait what??

Reply:
My defaults for projects ;) could remove it, if not needed